i3pystatus/pulseaudio/__init__.py

```diff
@@ -223,28 +220,33 @@ def sink_info_cb(self, context, sink_info_p, eol, _):
     def change_sink(self):
         sinks = list(s.split()[1] for s in self.sinks)
         if self.sink is None:
-            next_sink = sinks[(sinks.index(self.current_sink) + 1) %
-                              len(sinks)]
+            next_sink = sinks[(sinks.index(self.current_sink) + 1) % len(sinks)]
         else:
             next_sink = self.current_sink
 
         if self.move_sink_inputs:
-            sink_inputs = subprocess.check_output("pacmd list-sink-inputs".split(),
-                                                  universal_newlines=True)
+            sink_inputs = self._call_pactl(["list-sink-inputs"])
             for input_index in re.findall(r'index:\s+(\d+)', sink_inputs):
-                command = "pacmd move-sink-input {} {}".format(input_index, next_sink)
-
-                # Not all applications can be moved and pulseaudio, and when
-                # this fail pacmd print error messaging
-                with open(os.devnull, 'w') as devnull:
-                    subprocess.call(command.split(), stdout=devnull)
```

Comment:
This part of the code came from https://github.com/enkore/i3pystatus/pull/552. Since this PR uses the method `_call_pactl` instead that internally uses `subprocess.check_output`, the messages on stdout that could come from `pactl` will always be discarded.